reference/docs-conceptual/Migrating-from-Windows-PowerShell-5.1-to-PowerShell-7.md

```diff
@@ -0,0 +1,252 @@
+PowerShell looks in the specified directories when you do not specify a full path to a module. When
+starting a version of PowerShell, the `PSModulePath` reflects the module search path and only
+includes path segments appropriate for the version of PowerShell being started.
+
+[!NOTE] PowerShell sets the value of `$PSHOME\Modules` in the registry. It sets the value of
+`$HOME\Documents\PowerShell\Modules` each time you start PowerShell.
+
+The default value of Windows PowerShell `$Env:PSModulePath` on Windows is:
+
+```powershell
+$nv:PSModulePath -split (';')
+```
+
+```output
+C:\Users\<user>\Documents\WindowsPowerShell\Modules
+C:\Program Files\WindowsPowerShell\Modules
+C:\WINDOWS\System32\WindowsPowerShell\v1.0\Modules
+```
+
+The default value of PowerShell 7 `$Env:PSModulePath` on Windows combines the Windows PowerShell
+paths and the PowerShell 7 paths to provide autoloading of modules:
+
+```powershell
+$nv:PSModulePath -split (';')
+```
+
+```output
+C:\Users\<user>\Documents\PowerShell\Modules
+C:\Program Files\PowerShell\Modules
+C:\Program Files\PowerShell\7\Modules
+C:\Program Files\WindowsPowerShell\Modules
+C:\WINDOWS\System32\WindowsPowerShell\v1.0\Modules
+```
+
+[!NOTE] PowerShell 7 on Windows adds the module path for Windows PowerShell to permit the
+autoloading of modules from Windows PowerShell.
+
+### Starting Windows PowerShell from PowerShell 7
+
+PowerShell 7 on Microsoft Windows combines new module path locations needed for PowerShell 7 with
+those from Windows PowerShell to provide easy importing of both Core and Desktop modules. This
+includes adding or removing additional module paths created by applications or the user. When
+running Windows PowerShell from PowerShell 7, only the paths needed for Windows PowerShell modules
+are added.
+
+PowerShell 7 path on Windows before starting Windows PowerShell:
+
+```powershell
+$nv:PSModulePath -split (';')
+```
+
+```output
+C:\Users\<user>\Documents\PowerShell\Modules
+C:\Program Files\PowerShell\Modules
+C:\Program Files\PowerShell\7\Modules
+C:\Program Files\WindowsPowerShell\Modules
+C:\WINDOWS\System32\WindowsPowerShell\v1.0\Modules
+```
+
+Windows PowerShell path after starting `powershell` from PowerShell 7:
+
+```powershell
+powershell
+$nv:PSModulePath -split (';')
+```
+
+```output
+C:\Users\<user>\Documents\WindowsPowerShell\Modules
+C:\Program Files\WindowsPowerShell\Modules
+C:\WINDOWS\System32\WindowsPowerShell\v1.0\Modules
+```
+
+[!NOTE] Additional paths may exist if the user has added custom modules or applications that impact
+the path.
+
+For more information, see `PSModulePath` in [about_Environment_Variables](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7)
+
+for more information about Modules, see [about_Modules](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/7.0/Microsoft.PowerShell.Core/About/about_Modules.md)
```

Comment:
Does this para need to be extended in order to describe Win32 compatibility. NOT ALL System32 modules are even loadable in PWSH. I feel that should be mentioned here.